### ISSUE TITLE
Check fails with Python 3.7.2

### ISSUE THREAD

**Issue by kgizdov** [scikit-hep/root_numpy#384]
On ArchLinux with GCC 8.2.1, Python 3.7.2, when running:
```
nosetests -s -v root_numpy
```
I get the following errors:
```
======================================================================
ERROR: root_numpy.tests.test_evaluate.test_evaluate_func
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/gizdov/builds/python-root_numpy/src/root_numpy-4.8.0/root_numpy/tests/test_evaluate.py", line 43, in test_evaluate_func
    assert_raises(ValueError, rnp.evaluate, "f", arr_1d)
  File "/usr/lib/python3.7/unittest/case.py", line 743, in assertRaises
    return context.handle('assertRaises', args, kwargs)
  File "/usr/lib/python3.7/unittest/case.py", line 178, in handle
    callable_obj(*args, **kwargs)
  File "/home/gizdov/builds/python-root_numpy/src/root_numpy-4.8.0/root_numpy/_evaluate.py", line 105, in evaluate
    obj = ROOT.TFormula(uuid.uuid4().hex, obj)
TypeError: none of the 4 overloaded methods succeeded. Full details:
  TFormula::TFormula(const char* name, const char* formula = "", bool addToGlobList = true, bool vectorize = false) =>
    problem in C++; program state has been reset
  TFormula::TFormula(const char* name, const char* formula, int ndim, int npar, bool addToGlobList = true) =>
    takes at least 4 arguments (2 given)
  TFormula::TFormula() =>
    takes at most 0 arguments (2 given)
  TFormula::TFormula(const TFormula& formula) =>
    takes at most 1 arguments (2 given)

----------------------------------------------------------------------
Ran 227 tests in 2.944s

FAILED (errors=1)
```

**Comment by kratsg** [comment]
Which ROOT are you reporting this for?

**Comment by kgizdov** [comment]
ROOT is at v6.16.00

**Comment by kratsg** [comment]
Seems like they changed the TFormula interface here. Can you downgrade ROOT? if you seriously want to use this (support-dropped) package, you need to use 6.12-ish as the latest. Otherwise, if you want the newer ROOT, switch to uproot.

**Comment by kgizdov** [comment]
@kratsg if this package is deprecated, it should be reflected in the docs or something. I was unaware it's not longer in development.

**Comment by kratsg** [comment]
There's no active support -- or official maintainer. It's mostly held up by people volunteering their time now.

**Comment by eduardo-rodrigues** [comment]
The package is deprecated. Closing as won't do.
